pages/Documents/ConversationalAI/Third-PartyBots/AmazonLex/advanced-features.md

```diff
@@ -317,3 +317,79 @@ Once all of above steps has been configured and updated bot has been published t
 respond to the requests via the Amazon Lambda function. A demo of our WhatsApp map example (defined above) can be seen below:
 
 <img class="fancyimage" style="width:300px" src="img/lex/lex_richcontent_demo.gif">
+
+### Receiving Last consumer message (Messaging Only)
+
+Third-Party bot now provides a way to add the last consumer message as a part of [the welcome event](third-party-bots-amazon-lex-basic-content.html#the-welcome-event) (messaging only). When an ongoing conversation gets transferred to a new Agent or Skill, This enhancement will allow brands to respond to the last consumer message uttered as per their needs.
```

Comment:
i think you can remove `(messaging only)` in the description as it is already in the heading